src/scss/skin-modern/components/_replaybutton.scss

```diff
@@ -0,0 +1,12 @@
+@import '../variables';
+@import '../mixins';
+
+.#{$prefix}-ui-replaybutton {
+  @extend %ui-button;
+
+  background-image: svg('assets/skin-modern/images/prev.svg');
```

Comment:
I guess this is the same Image as our large replay icon at the end of playback?

Reply:
No it's not, and on purpose. The large replay icon put in the small controlbar looks very strange as 
a) it's hard to see due to the circle around it and so on, and 
b) due to it's design also doesn't fit to other icons like the playbacktogglebutton

Reply:
If you think we should use something more similar to the hugereplaybutton I'm happy to talk to our designers for that.

Reply:
I was just asking because I did not see this `svg` added in the PR. 

But I trust your (and design) judgement on this one. 

As long as the points you mentioned are fulfilled it is fine.

Reply:
Added new icon in cb24b06c.
Preview: 
![image](https://user-images.githubusercontent.com/4820162/203820757-3f56c30a-c3c6-47fd-b6c2-de387f9a7169.png)

Reply:
Thanks! Looks fitting! 

I personally would not place it beside the play-button to avoid hitting the replay when wanting to pause the content, but I guess that is up to the customization then. 

Icon looks fine.

Reply:
Yes, placement of the button is up to the customer of course, this is just an example to show in the PR how it looks like.